org.uqbar.project.wollok.typeSystem/src/org/uqbar/project/wollok/typesystem/TypeSystemWollokValidatorExtension.xtend

```diff
@@ -14,21 +15,10 @@ import org.uqbar.project.wollok.wollokDsl.WFile
  */
 class TypeSystemWollokValidatorExtension implements WollokValidatorExtension {
 
-	@Check(NORMAL)
+	@Check(FAST)
```

Comment:
Now that the type analysis runs in the builder, the "validation" here does not take time, it only informs errors previously computed.